src/main/java/io/cdap/e2e/pages/actions/CdfGcsActions.java

```diff
@@ -147,6 +167,11 @@ public static void enterDelimiterField(String delimiter) {
     CdfGCSLocators.delimiterField.sendKeys(delimiter);
   }
 
+  /**
+   * @deprecated
+   * Use either {@link io.cdap.e2e.utils.CdfHelper#openSinkPluginPreviewData(String)}
+   * or {@link io.cdap.e2e.utils.CdfHelper#openSourcePluginPreviewData(String)} as per plugin type.
+   */
   public static void clickPreviewData() {
```

Comment:
Add @Deprecated annotation here and elsewhere

Reply:
Added annotations